exe/launcher/Main.hs

```diff
@@ -118,16 +123,16 @@ nodeHttpBridgeOn stateDir port net =
         ] ++ networkArg
     networkArg = maybe [] (\d -> ["--networks-dir", d]) stateDir
 
-walletOn :: Maybe FilePath -> Port "Wallet" -> Port "Node" -> String -> Command
+walletOn :: Maybe FilePath -> Maybe (Port "Wallet") -> Port "Node" -> String -> Command
```

Comment:
This line is now too long (https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters).

Suggest: 
```suggestion
walletOn
    :: Maybe FilePath
    -> Maybe (Port "Wallet")
    -> Port "Node"
    -> String
    -> Command
```